<ui version="4.0" >
 <class>circleMotionGenerator</class>
 <widget class="QWidget" name="circleMotionGenerator" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>146</width>
    <height>116</height>
   </rect>
  </property>
  <property name="maximumSize" >
   <size>
    <width>16777215</width>
    <height>2332232</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" >
   <property name="spacing" >
    <number>1</number>
   </property>
   <property name="leftMargin" >
    <number>1</number>
   </property>
   <property name="topMargin" >
    <number>1</number>
   </property>
   <property name="rightMargin" >
    <number>1</number>
   </property>
   <property name="bottomMargin" >
    <number>1</number>
   </property>
   <item>
    <layout class="QVBoxLayout" >
     <property name="spacing" >
      <number>1</number>
     </property>
     <item>
      <layout class="QGridLayout" >
       <property name="horizontalSpacing" >
        <number>1</number>
       </property>
       <property name="verticalSpacing" >
        <number>1</number>
       </property>
       <item row="0" column="0" colspan="2" >
        <widget class="QComboBox" name="motionSelect" />
       </item>
       <item row="1" column="0" colspan="2" >
        <widget class="QGroupBox" name="groupBox" >
         <property name="title" >
          <string>Плоскость движения</string>
         </property>
         <layout class="QGridLayout" >
          <property name="leftMargin" >
           <number>1</number>
          </property>
          <property name="topMargin" >
           <number>1</number>
          </property>
          <property name="rightMargin" >
           <number>1</number>
          </property>
          <property name="bottomMargin" >
           <number>1</number>
          </property>
          <property name="horizontalSpacing" >
           <number>1</number>
          </property>
          <property name="verticalSpacing" >
           <number>1</number>
          </property>
          <item row="0" column="0" >
           <widget class="QRadioButton" name="oxzButton" >
            <property name="text" >
             <string>Oxz</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QRadioButton" name="oyzButton" >
            <property name="text" >
             <string>Oyz</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2" >
           <widget class="QRadioButton" name="oxyButton" >
            <property name="text" >
             <string>Oxy</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0" colspan="2" >
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>Радиусы эллипса</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" >
        <widget class="QDoubleSpinBox" name="radius1Edit" />
       </item>
       <item row="3" column="1" >
        <widget class="QDoubleSpinBox" name="radius2Edit" />
       </item>
      </layout>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>289</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
